jetty-nosql/src/main/java/org/eclipse/jetty/nosql/mongodb/MongoSessionIdManager.java

```diff
@@ -179,55 +184,60 @@ public MongoSessionIdManager(Server server, DBCollection sessions)
                 BasicDBObjectBuilder.start().add("id",1).get(),
                 BasicDBObjectBuilder.start().add("unique",true).add("sparse",false).get());
         _sessions.ensureIndex(
-                BasicDBObjectBuilder.start().add("id",1).add("version",1).get(),
-                BasicDBObjectBuilder.start().add("unique",true).add("sparse",false).get());
+                BasicDBObjectBuilder.start().add("id", 1).add("version", 1).get(),
```

Comment:
Sorry about that, apparently IntelliJ 14 changed the way it strips white space. Didn't notice it until I put in the pull request. I can try and back those out if you like.